scipy/special/add_newdocs.py

```diff
@@ -161,6 +161,62 @@ def add_newdoc(place, name, doc):
 
     """)
 
+
+add_newdoc("scipy.special", "agm",
+    """
+    agm(a, b)
+
+    Compute the arithmetic-geometric mean of `a` and `b`.
+
+    Start with a_0 = a and b_0 = b and iteratively compute::
+
+        a_{n+1} = (a_n + b_n)/2
```

Comment:
Yikes we've certainly violated that convention a bunch in special; e.g. randomly clicking on some functions turns up:

- https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.ellipkm1.html#scipy.special.ellipkm1
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.itmodstruve0.html#scipy.special.itmodstruve0
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.gdtrc.html#scipy.special.gdtrc

and so on.

Could we use unicode in the short definitions? Then we could have symbols like ∫ and Σ and get nice-looking text definitions. (Not relevant for `agm` of course, just thinking out loud.)